### PR TITLE
read me updates with website integration

### PR DESCRIPTION
**Pull request by Resseguie**
incorporates and links to official johnny-five website

### CODE REVIEW

**Comment by Resseguie** [comment]
@rwaldron this is a bit of content addition and re-organization that incorporates links to the official website as well as some content from your announcement blog. (Links to specific pages, etc.)

File: README.md [hunk 1]
```diff
@@ -68,7 +81,28 @@ Johnny-Five does not attempt to provide "all the things", but instead focuses on
...
+
+board.on("ready", function() {
+  // Create an Led on pin 13
+  var led = new five.Led(13);
+  // Blink every half second
+  led.blink(500); 
+});
+```
+
+<img src="https://github.com/rwaldron/johnny-five/raw/master/assets/led-blink.gif">
```
Comment (Resseguie) [inline README.md]: Will show up as broken until the image with this commit is added...

File: tpl/.readme.md [hunk 2]
```diff
@@ -1,13 +1,26 @@
 <img src="https://github.com/rwaldron/johnny-five/raw/master/assets/sgier-johnny-five.png">
 
-# Node-isassemble Johnny-Five
```
Comment (rwaldron) [inline tpl/.readme.md]: Sad to see it go, but I think you're right.

Comment (Resseguie) [inline tpl/.readme.md]: Johnny's all grown up?

**Pull request merged**